src/Evolution/Systems/Cce/System.hpp

```diff
@@ -0,0 +1,116 @@
+ * computing the inputs to the hypersurface equations are
+ * `Cce::PrecomputeCceDependencies` (see template specializations),
+ * `Cce::mutate_all_precompute_cce_dependencies`, `Cce::PreSwshDerivatives` (see
+ * template specializations), `Cce::mutate_all_pre_swsh_derivatives_for_tag`,
+ * and `Cce::mutate_all_swsh_derivatives_for_tag`. There are a number of
+ * typelists in `IntegrandInputSteps.hpp` that determine the set of quantities
+ * to be evaluated in each of the five hypersurface steps.
+ *
+ * Once the hypersurface equation inputs are computed, then a hypersurface
+ * equation right-hand side can be evaluated via `Cce::ComputeBondiIntegrand`
+ * (see template specializations). Then, the hypersurface equation may be
+ * integrated via `Cce::RadialIntegrateBondi` (see template specializations).
+ * Note that some of the hypersurface equations are not of simple form, and
+ * therefore require more careful techniques than a direct indefinite radial
+ * integration.
+ *
+ * Relevant parts of the parallel infrastructure for performing the hypersurface
+ * steps are: `Cce::CharacteristicEvolution`,
+ * `Cce::Actions::CalculateIntegrandInputsForTag`, and
+ * `Cce::Actions::PrecomputeGlobalCceDependencies`. Note that most of the
+ * algorithmic steps are laid out in order in the phase-dependent action list of
+ * `Cce::CharacteristicEvolution`.
+ *
+ * At this point, all of the Bondi quantities on a given hypersurface have been
+ * evaluated, and we wish to output the relevant waveform quantities at
+ * \f$\mathcal I^+\f$. This acts much like an additional step in the
+ * hypersurface sequence, with inputs that need to be calculated before the
+ * quantities of interest can be evaluated. The action
+ * `Cce::Actions::CalculateScriInputs` performs the sequence of steps to obtain
+ * those inputs, and the utilities `Cce::CalculateScriPlusValue` (see template
+ * specializations) can be used to evaluate the desired outputs at
+ *\f$\mathcal I^+\f$.
+ *
+ * Unfortunately, those quantities at \f$\mathcal I^+\f$ are not yet an
+ * appropriate waveform output, because the time coordinate with which they are
+ * evaluated is the simulation time, not an aymptotically inertial time. So,
+ * instead of directly writing the waveform outputs, we must put them in a queue
+ * to be interpolated once enough data points have been accumulated to perform a
+ * reliable interpolation at a consistent cut of \f$I^+\f$ at constant inertial
+ * time. Utilities for calculating and evolving the asymptotic inertial time are
+ * `Cce::InitializeScriPlusValue` and `Cce::CalculateScriPlusValue` using
+ * arguments involving `Cce::Tags::InertialRetardedTime`. A utility for managing
+ * the interpolation is `Cce::ScriPlusInterpolationManager`, and relevant parts
+ * of the parallel infrastructure for manipulating the data into the
+ * interpolator and writing the results to disk are
+ * `Cce::Actions::EnqueueInterpolationScriData` and
```

Comment:
Cce::Actions::EnqueueInterpolationScriData is not a link, (and didn't we ask for a different name earlier?)